harness/determined/common/experimental/determined.py

```diff
@@ -124,8 +124,9 @@ def logout(self) -> None:
 
         authentication.logout(self._session._master, user, self._session._cert)
 
-    def list_users(self) -> Sequence[user.User]:
-        users_bindings = bindings.get_GetUsers(session=self._session).users
+    def list_users(self, all_users: bool = False) -> Sequence[user.User]:
```

Comment:
I'm not sure if I should break the SDK here

I do think parity between list_users and CLI makes sense?

Reply:
I think adding an optional parameter in our py SDK would be pretty low on the list of what would be a breaking change.
@azhou-determined @wes-turner

Reply:
i think the feature makes sense as an add to the SDK, but i'm not sold on the naming. if this is just a filter on the "active" property of the user, maybe something like `include_active` or `active`? would be more explicit.

Reply:
I think `include_inactive` makes sense for the REST API, too. It's nice when booleans are named such that they're clearly boolean.

Reply:
Changed SDK and CLI to use ```include_inactive```, kept API as optional parameter of ```active``` to have 3 states if the webui eventually wants to filter on it

On the breaking change, the breaking change is not adding the optional parameter rather making the optional parameter default to ```include_inactive=False``` to try and keep parity with the CLI and hide inactive users by default. Do you think that makes sense or we could set ```include_inactive=True``` on the SDK by default to avoid the breakage?

Reply:
Awesome. Thanks for making that change. I do like your `include_inactive=True` suggestion for maintaining consistent behavior, but if excluding inactive users from is a thing that our customers should be doing to make their own lives easier, could we run this by a product owner? Maybe @hkumar92 as the product representative for RBAC?